includes/admin/upgrades/v3/class-data-migrator.php

```diff
@@ -970,26 +979,34 @@ public static function orders( $data = null ) {
 					continue;
 				}
 
-				edd_add_order_adjustment( array(
-					'object_id'   => $order_id,
-					'object_type' => 'order',
-					'type_id'     => $discount->id,
-					'type'        => 'discount',
-					'description' => $discount,
-					'subtotal'    => $subtotal - $discount->get_discounted_amount( $subtotal ),
-					'total'       => $subtotal - $discount->get_discounted_amount( $subtotal ),
-				) );
+				edd_add_order_adjustment(
+					array(
+						'object_id'     => $order_id,
+						'object_type'   => 'order',
+						'type_id'       => $discount->id,
+						'type'          => 'discount',
+						'description'   => $discount,
+						'subtotal'      => $subtotal - $discount->get_discounted_amount( $subtotal ),
+						'total'         => $subtotal - $discount->get_discounted_amount( $subtotal ),
+						'date_created'  => $date_created_gmt,
```

Comment:
Should this be `$data->post_modified_gmt`? (If not, does it deserve an inline comment because it's different?)